### PR TITLE
Passed all tests and fixed tests

### PR DESCRIPTION
**Pull request by ghost1497**
Ok google, what is a fire drill.

### CODE REVIEW

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java [hunk 1]
```diff
@@ -42,31 +44,61 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+        boolean flick = false;
```
Comment (Git-Leon) [inline src/main/java/com/zipcodewilmington/StringArrayUtils.java]: > Complex is better than complicated

```java
return getNumberOfOccurrences(array, value) > 0
```

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java [hunk 2]
```diff
@@ -42,31 +44,61 @@ public static String getSecondToLastElement(String[] array) {
...
+        return revArray;
     }
 
     /**
      * @param array array of String objects
      * @return true if the order of the array is the same backwards and forwards
      */ // TODO
     public static boolean isPalindromic(String[] array) {
-        return false;
+        String[] revArray = reverse(array);
```
Comment (Git-Leon) [inline src/main/java/com/zipcodewilmington/StringArrayUtils.java]:
```java
return Arrays.equals(array, reverse(array));
```

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java [hunk 3]
```diff
@@ -84,23 +122,76 @@ public static int getNumberOfOccurrences(String[] array, String value) {
      * @return array with identical contents excluding values of `value`
      */ // TODO
     public static String[] removeValue(String[] array, String valueToRemove) {
-        return null;
+        int occurances = getNumberOfOccurrences(array, valueToRemove);
```
Comment (Git-Leon) [inline src/main/java/com/zipcodewilmington/StringArrayUtils.java]: 👍

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java (hunk 2 above)
Comment (ghost1497) [reply src/main/java/com/zipcodewilmington/StringArrayUtils.java]: 🥅

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java (hunk 1 above)
Comment (ghost1497) [reply src/main/java/com/zipcodewilmington/StringArrayUtils.java]: 🎣

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java (hunk 3 above)
Comment (ghost1497) [reply src/main/java/com/zipcodewilmington/StringArrayUtils.java]: 👍

**Pull request closed without merging**